tests/ops/test_simple_gla.py

```
@@ -254,6 +252,153 @@
+@pytest.mark.parametrize('B', test_b_list)
+@pytest.mark.parametrize('T', test_t_list)
+@pytest.mark.parametrize('H', test_h_list)
+@pytest.mark.parametrize('D', test_d_list)
+@pytest.mark.parametrize('gate_logit_normalizer', test_gate_list)
+@pytest.mark.parametrize('dtype', [torch.float16])
+@pytest.mark.parametrize('scale', [1, 0.1])
+@pytest.mark.skipif(
+    os.getenv('SKIP_TEST_CHUNK_VARLEN') == '0',
+    reason='Skipping test because TEST_CHUNK_VARLEN is enabled'
+)
+def test_fused_chunk(
```

Pylint (3.3.7)

[refactor] Too many arguments (7/5)

(R0913)